index.html

```diff
@@ -0,0 +1,353 @@
+
+### Monitor
+
+A <dfn>monitor</dfn> is an output device that displays information in pictorial form. A monitor usually comprises the display, circuitry, casing, and power supply. ([Wikipedia](https://en.wikipedia.org/wiki/Computer_monitor))
+
+### Display
+
+A <dfn>display</dfn>, informally a [=screen=], is a display device for presentation of images, text, or video transmitted electronically, without producing a permanent record. Electronic visual displays include television sets, computer monitors, and digital signage. ([Wikipedia](https://en.wikipedia.org/wiki/Electronic_visual_display))
+
+### Screen
+
+<dfn>Screen</dfn> is a synonym to [=display=]. [=Screen=] is used more in software context (logical) while display is primarily used in hardware context (physical). See also the [=Screen interface=].
+
+### Multiple internal display
+
+A <dfn>multiple internal display</dfn> (MID) device is a more technical synonym to a [=dual-screen device=].
+
+### Single foldable display
+
+A <dfn>single foldable display</dfn> (SFD) refers to a [=foldable-screen device=] with a single screen.
+
+### Posture
+
+A <dfn>posture</dfn> is defined as the primary way in which the [=foldable device=] is positioned and presented to the user. Because [=foldable devices=] have a hinge they can present different [=postures=] depending on the hinge angle, device orientation, and the way on how the device is being held by the user. Example postures include “book”, “flat”, “laptop”, and “tent”. The list of postures is expected to evolve over time, and not all foldable devices are able to support all possible postures.
+
+### Window segments
+
+A <dfn>window segment</dfn> represents the regions (and their dimensions) of the window that reside on separate (adjacent) [=displays=].
+
+### Spanning
+
+<dfn>Spanning</dfn> is a concept that allows apps or websites to run side by side on a [=dual-screen device=], spanning web content across [=window segments=]. The ability to maximize a given application across the two [=screens=] is enabled through the [=window manager=]. Because spanning across the two screens double the screen estate it is expected that applications and websites leverage that additional space to optimize or enhance the UX to provide a better experience.
```

Comment:
The first sentence isn't quite right, perhaps this paragraph needs some reworking. Foldables allow apps to run "side by side", but spanning itself is not that.

<dfn>Spanning</dfn> is a window mode on a [=dual-screen device=], where a single application or web content window is spread across both screens of the device. The ability to span a given application across the two [=screens=] is enabled through the [=window manager=]. Applications and websites may wish to optimize their layout, incorporating the physical gap of the [=dual-screen device=] hinge, to provide a better experience.

Reply:
+1ish . I agree that spanning in a PC-like scenario generally means across displays, but I think we need to be careful to allow 'spanning' to mean _across the border_ (whether this border is a display edge or a hinge or a screen). 

 The Screen Fold API can create beautiful innovative layouts with the help of the CSS spanning features on a Galaxy Fold for example, even though it doesn't have 2 screens, and the main concept here is that the developer can get some content to span or dock it alongside the hinge. in this case the spanning is across the hinge.

Spanning is more powerful if abstracted to a border than to a screen?

what are we spanning across, might be the question. Are we getting too philosophical here?  

to span or not to span... that is the question.

Reply:
I adopted the @michaelwasserman's proposal for the first sentence and the note proposed by @diekus.